exercises/practice/isbn-verifier/.meta/exercise-data.yaml

```diff
@@ -0,0 +1,31 @@
+properties:
+  isValid:
+    test: |-
+      sprintf(q:to/END/, %case<input><isbn>, %case<expected>, %case<description>.raku);
+      cmp-ok(
+          is-isbn10('%s'),
```

Comment:
Using `'` to quote is likely fine, but there is the possibility of a new test case including a quote which would cause problems with this. I'd say its typically better to use the `.raku` method on string inputs so any necessary escaping is handled automatically.

Reply:
I have modified it to use `.raku`. I appreciate the feedback and insight.
8-)